tests/t520-operator.c

```diff
@@ -0,0 +1,268 @@
+/// @file
+/// Test creation creation, action, and destruction for mass matrix operator
+/// \test Test creation creation, action, and destruction for mass matrix operator
+#include <ceed.h>
+#include <stdlib.h>
+#include <math.h>
+#include "t310-basis.h"
+
+/* The mesh comprises of two rows of 3 quadralaterals followed by one row
+     of 6 triangles:
+   _ _ _
+  |_|_|_|
+  |_|_|_|
+  |/|/|/|
+
```

Comment:
This example is on a mixed mesh - We can also support mapping the same dofs to different quadrature points, as would be helpful for the wind turbine examples